src/test/kotlin/com/zepben/evolve/cim/iec61968/assetinfo/WireMaterialKindTest.kt

```diff
@@ -21,7 +21,7 @@ internal class WireMaterialKindTest {
 
     @Test
     internal fun validateVsPb() {
-        validateEnum(WireMaterialKind.values(), PBWireMaterialKind.values())
+        validateEnum(WireMaterialKind.entries, PBWireMaterialKind.entries)
```

Comment:
Changed away from deprecated function. Also changed in all other enum validations.